src/tasks/tasks.py

```diff
@@ -2092,11 +2081,11 @@ def _stream_records(self, filename):
     def load_data(self):
         iters_by_split = collections.OrderedDict()
         for split, filename in self._files_by_split.items():
-            iter = list(self._stream_records(filename))
-            iters_by_split[split] = iter
+            if "test" in filename:
```

Comment:
This seems brittle. What if someone tries to evaluate on a span task with a public test set?

Reply:
This still seems dangerously brittle...